src/bgw/job_stat.c

```diff
@@ -800,21 +800,21 @@ ts_bgw_job_stat_next_start(BgwJobStat *jobstat, BgwJob *job, int32 consecutive_f
 		/* Update the errors table regarding the crash */
 		if (!ts_flags_are_set_32(jobstat->fd.flags, LAST_CRASH_REPORTED))
 		{
-			NameData proc_schema = { .data = { 0 } }, proc_name = { .data = { 0 } };
-			JsonbParseState *parse_state = NULL;
-			JsonbValue *result = NULL;
+			// NameData proc_schema = { .data = { 0 } }, proc_name = { .data = { 0 } };
```

Comment:
This function contains some commented-out lines. Do you plan to remove them in the final PR?

Reply:
It was some leftovers! Removed!